Update documentation to install binaries and connect to multiple hosts (#443)
osrf/rmw_zenoh: README.md

> Note: See available distro branches, eg. `jazzy`, for supported ROS 2 distributions.

## Installation

> Options for installing `rmw_zenoh`

Reviewer: ```suggestion
`rmw_zenoh` can either be installed via binaries (recommended for stable development) or from source (recommended if latest features are needed). See instructions below.
```

> ### Binary Packages

Reviewer: ```suggestion
### Binary Installation
```

Binary packages are available for Tier-1 operating systems. Install `ros-<DISTRO>-rmw-zenoh-cpp`. Replace

> `<DISTRO>` with the ROS distro into which the packages should be installed.

Reviewer: ```suggestion
Binary packages for supported ROS 2 distributions (see available distro branches) are available on respective [Tier-1](https://www.ros.org/reps/rep-2000.html#support-tiers) platforms for the distributions. First ensure that you system is set up to install ROS 2 binaries by following the instructions [here](https://docs.ros.org/en/rolling/Installation/Ubuntu-Install-Debs.html).

Then install `rmw_zenoh` binaries using the command

```bash
sudo apt update && sudo apt install ros-<DISTRO>-rmw-zenoh-cpp # replace <DISTRO> with codename for the distribution, eg., rolling
```

> ### Building rmw_zenoh_cpp from source

Reviewer: ```suggestion
### Source Installation
```

```

### Connecting multiple hosts
By default, all discovery traffic is local per host, where the host is the computer running a `Zenoh router`,
and each host in a multiple-host system runs a `Zenoh router`.
To bridge communications across two hosts, the `Zenoh router` configuration for one of the hosts must be
updated to connect to the other host's `Zenoh router` at startup.
This is done by setting the `ZENOH_ROUTER_CONFIG_URI` environment variable in one of the hosts to point to a
modified `Zenoh router` configuration file which specifies an endpoint that the other host's `Zenoh router` is

> is listening on. `Zenoh routers` print a list of endpoints they listen on when they start - choose a suitable one.

Reviewer: Nit: The printout would be subjective of the log level set. 
```suggestion
is listening on. 
```

> Make a copy of rmw_zenoh_cpp/config/DEFAULT_RMW_ZENOH_ROUTER_CONFIG.json5 and modify the section that begins:

Reviewer: ```suggestion
Make a copy of [DEFAULT_RMW_ZENOH_ROUTER_CONFIG.json5]( rmw_zenoh_cpp/config/DEFAULT_RMW_ZENOH_ROUTER_CONFIG.json5) and modify the `connect` block to include the endpoint of the other `Zenoh router`. For example, if the other `Zenoh router` is listening on IP address `192.168.1.1` and port `7447` on its host, modify config file to connect to this router as shown below:
```

> ```json5
> mode: "router",
> ```
> as seen below.
> In this example, the `Zenoh router` will connect to the `Zenoh router` running on a second host with IP address `192.168.1.1` and port `7447`.

Reviewer: ```suggestion
```
